testsuite/features/step_definitions/api_common.rb

```diff
@@ -479,7 +479,7 @@ def wait_action_complete(actionid, timeout: DEFAULT_TIMEOUT)
   channel = if arch != 'x86_64'
               'test-channel-i586'
             else
-              'test-channel-x86_64'
+              'test-channel-for-sle'
```

Comment:
Not really fun with this label name, as I said above, I think it will be better to have a label name, matching the channel name, and including then x86_64 word on it.